pkg/cvo/updatepayload.go

```diff
@@ -38,21 +38,25 @@ func (optr *Operator) defaultPayloadDir() string {
 
 func (optr *Operator) defaultPayloadRetriever() PayloadRetriever {
 	return &payloadRetriever{
-		kubeClient:   optr.kubeClient,
-		operatorName: optr.name,
-		releaseImage: optr.release.Image,
-		namespace:    optr.namespace,
-		nodeName:     optr.nodename,
-		payloadDir:   optr.defaultPayloadDir(),
-		workingDir:   targetUpdatePayloadsDir,
-		verifier:     optr.verifier,
+		kubeClient:           optr.kubeClient,
+		operatorName:         optr.name,
+		releaseImage:         optr.release.Image,
+		namespace:            optr.namespace,
+		nodeName:             optr.nodename,
+		payloadDir:           optr.defaultPayloadDir(),
+		workingDir:           targetUpdatePayloadsDir,
+		verifier:             optr.verifier,
+		verifyTimeoutOnForce: 2 * time.Minute,
+		downloadTimeout:      2 * time.Minute,
 	}
 }
 
 const (
 	targetUpdatePayloadsDir = "/etc/cvo/updatepayloads"
 )
 
+type downloadFunc func(context.Context, configv1.Update) (string, error)
```

Comment:
Possibly grow some godocs here, like:

> downloadFunc downloads the requested update, and returns either a path on the local filesystem containing extracted manifests or an error.

or some such, so folks don't wonder "what's that `string` return about?".

Reply:
:+1: good point

Reply:
I've sprinkled several more godocs when at it